library/Icinga/Crypt/AesCrypt.php

```diff
@@ -50,22 +57,18 @@ class AesCrypt
     /** @var string The authentication tag which is passed by reference when using AEAD cipher mode */
     private $tag;
 
-    /** @var string The cipher method */
-    private $method = 'AES-128-GCM';
+    private $method = 'aes-256-gcm';
 
     const GCM_SUPPORT_VERSION = '7.1';
 
     public function __construct($random_bytes_len = 128)
     {
-        if (version_compare(PHP_VERSION, self::GCM_SUPPORT_VERSION, '<')) {
-            $this->method = 'AES-128-CBC';
-        }
-
+        $this->method = $this->getSupportedMethod();
```

Comment:
Didn't I suggest to put this into the getter for the cipher?

The reason why is that constructors should perform as less work as possible. Also, if someone uses `setMethod` first, it won't run unnecessarily.